react/features/base/redux/readme.md

```diff
@@ -1,30 +1,24 @@
 Jitsi Meet - redux state persistency
 ====================================
-Jitsi Meet has a persistency layer that persist a subtree (or specific subtrees) into window.localStorage (on web) or
+Jitsi Meet has a persistency layer that persists a subtree (or specific subtrees) into window.localStorage (on web) or
 AsyncStorage (on mobile).
 
 Usage
 =====
-If a subtree of the redux store should be persisted (e.g. ``'features/base/participants'``), then persistency for that
-subtree should be enabled in the config file by creating a key in
+If a subtree of the redux store should be persisted (e.g. ``'features/base/profile'``), then persistency for that
+subtree should be requested by registering the subtree (and related config) into PersistencyRegistry.
 
+E.g. to register the field ``profile`` of the Redux subtree ``'features/base/profile'`` to be persisted, use:
```

Comment:
How about nested objects?

Reply:
Can you please show/give an example?

Reply:
If you have a feature which has several keys, one of which is an object, and you want to persist only part of that.

```
{
   foo: "foo",
   bar: {
        baz: "xxxx",
        secret: 1234
   }
}
```

How do we persist this without including the "secret" key?

Reply:
that seems to me as a new feature which we would add when needed

Reply:
also in order to avoid such situation we could branch the object, so that it would have secret under other root key

Reply:
Why design something limited from the start? Passing a function à la Array.filter could be a solution.

Reply:
what's the point of adding unused code ?

Reply:
or maybe I just don't get what you mean exactly by "passing function a la Array.filter"...

Reply:
Same here, don't really understand neither the Array.filter reference, nor why would we add a feature that may not be used in the next 1-2-6-7 months, just to find out later when we try to use it, it's already broken.